pkgs/database/src/lib.zig

```diff
@@ -0,0 +1,5 @@
+const rocksDB = @import("./rocksdb.zig");
+pub usingnamespace rocksDB;
```

Comment:
`usingnamespace` already removed in 0.15, it is better to not use it in new codes

Reply:
+1, can we do a cleanup followup PR on this (and other usingnamespace uses) cc @noopur23 create an issue for this